lib/src/bluez_client.dart

```diff
@@ -1941,11 +238,11 @@ class BlueZClient {
     return BlueZDevice(this, object);
   }
 
-  bool _isDevice(_BlueZObject object) {
+  bool _isDevice(BlueZObject object) {
     return object.interfaces.containsKey('org.bluez.Device1');
   }
 
-  List<BlueZGattService> _getGattServices(DBusObjectPath parentPath) {
```

Comment:
This was previously private API that has now been exposed. Is there a suitable technique for keeping this private?